UnitsNet/GeneratedCode/Quantities/Acceleration.NetFramework.g.cs

```diff
@@ -836,6 +836,16 @@ public string ToString([CanBeNull] IFormatProvider provider, [NotNull] string fo
         /// </summary>
         /// <param name="format">The format string.</param>
         /// <returns>The string representation.</returns>
+        /// <remarks>
+        /// The valid format strings are as follows:
+        /// "g": The quantity formatted as the value with 2 significant digits after the radix, and abbreviation. For example: 1.23 m
```

Comment:
Can remove "The quantity formatted as", it's implicit in the context.

Maybe something like this: 
`The value with 2 significant digits after the radix followed by the unit abbreviation, such as "1.23 m".`